webui/src/pages/bug/CommentForm.tsx

```diff
@@ -89,6 +91,7 @@ function CommentForm({ bugId }: Props) {
           onChange={(comment: string) => setIssueComment(comment)}
         />
         <div className={classes.actions}>
+          <CloseBugButton bugId={bugId} />
```

Comment:
You need more state here. The "Close" button should only appear when the bug is open. If the bug is closed, you should have a "Reopen" button instead.

Reply:
For sure. If the bug is closed I will just make the "Close" button disabled in this issue.
I´ll implement the Reopen action in another issue.

Reply:
Done!